flutter/test/default_integrations_test.dart

```diff
@@ -264,3 +290,21 @@ class Fixture {
   final hub = MockHub();
   final options = SentryFlutterOptions();
 }
+
+class LoadReleaseIntegrationFixture {
```

Comment:
this test class already has a `Fixture` class, you just need to adapt that it'd be suitable for every test, not creating a Fixture class per "test", a better example would be java and ios repos
eg https://github.com/getsentry/sentry-java/blob/main/sentry/src/test/java/io/sentry/TracesSamplerTest.kt
or just search across this repo per `Fixture`